data/remnant/remnant 1 introduction.txt

```diff
@@ -3717,6 +3735,7 @@ mission "Remnant: Expanded Horizons Storms 1"
 	passengers 1
 	source
 		government "Remnant"
+		attributes "remnant primary"
```

Comment:
```suggestion
```

There are plenty of hallways on both Esquiline and Ssil Vida. There's no need to restrict where she might appear for this mission.

Reply:
It might be useful for you to comment on #7026 if you don't think that's actually a problem.